docs/admin/kube-apiserver.md

```diff
@@ -78,7 +73,8 @@ kube-apiserver
       --etcd-servers=[]: List of etcd servers to connect with (http://ip:port), comma separated.
       --etcd-servers-overrides=[]: Per-resource etcd servers overrides, comma separated. The individual override format: group/resource#servers, where servers are http://ip:port, semicolon separated.
       --event-ttl=1h0m0s: Amount of time to retain events. Default 1 hour.
-      --experimental-keystone-url="": If passed, activates the keystone authentication plugin
+      --experimental-keystone-auth-mode="": If passed, selects between token and password modes
+      --experimental-keystone-config="": If passed, activates the keystone authentication plugin
```

Comment:
Need some more details like configuration file needs to be passed. If possible lets talk about config file details in doc section.